### PR TITLE
mention slack callback_id

### PR DESCRIPTION
**Pull request by auhlig**
Signed-off-by: Arno Uhlig <arno.uhlig@sap.com>

Update docs for slack `callback_id` after https://github.com/prometheus/alertmanager/pull/1592 is merged

### CODE REVIEW

**Comment by simonpasquier** [comment]
@brian-brazil PTAL

File: content/docs/alerting/configuration.md [hunk 1]
```diff
@@ -503,6 +503,7 @@ actions:
   [ <action_config> ... ]
 [ color: <tmpl_string> | default = '{{ if eq .Status "firing" }}danger{{ else }}good{{ end }}' ]
 [ fallback: <tmpl_string> | default = '{{ template "slack.default.fallback" . }}' ]
+[ callback_id: <tmpl_string> | default = '{{ template "slack.default.callbackid" . }}' ]
```
Comment (brian-brazil) [inline content/docs/alerting/configuration.md]: Keep it sorted

Comment (auhlig) [reply content/docs/alerting/configuration.md]: sorted

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**